packages/blockchain-wallet-v4-frontend/src/data/components/exchange/sagas.utils.ts

```diff
@@ -88,6 +88,8 @@ export default ({ coreSagas, networks }) => {
         .build()
         .done()).value()
     } catch (e) {
+      // eslint-disable-next-line
+      console.log(e)
```

Comment:
Why'd you want to log the error here?

Reply:
This usually ends up being where an error happens and it goes unnoticed because we `return {}`. It would probably be better to actually throw the error but I dont want to break existing code somewhere else :/